.github/workflows/cleanup_caches.yml

```diff
@@ -34,15 +36,16 @@ jobs:
           echo "Cache info: "
           du -h -d2 ${PIP_CACHE_PATH}
 
-  Cleanup_CCACHE:
+  Cleanup_ccache_lin:
+    name: Cleanup Linux ccache
     runs-on: aks-linux-2-cores-8gb
     if: ${{ github.repository_owner == 'openvinotoolkit' }}
     container:
       image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
       volumes:
         - /mount:/mount
     env:
-      CCACHE_PATH: /mount/caches/ccache/ubuntu22_riscv64
+      CCACHE_PATH: /mount/caches/ccache
```

Comment:
@ababushk, it points to the Blob Storage, right? Isn't it supposed to cleanup old blobs by itself, or...?